### ISSUE TITLE
[docs] Installing expo-updates in Bare Workflow with expo-modules

### ISSUE THREAD

**Issue by homosape** [expo/expo#20570]
### Summary

Please update[ this documentation](https://github.com/expo/expo/tree/e5cc03bf5ec3cdd50f059301c247b690173ab181/packages/expo-updates) regarding implementing `expo-updates` into bare workflow 

The guidelines there use `react-native-unimodules` but Expo has migrated to `expo-modules` and those docs did not reflect the changes. 

**Especially, the line where it asks if project uses** [expo-splash-screen](https://github.com/expo/expo/tree/e5cc03bf5ec3cdd50f059301c247b690173ab181/packages/expo-updates#expo-splash-screen)

The issue which I linked up there regarding installing `expo-updates` with `expo-splash-screen` is related to a notorious bug **"No native splash screen registered for given view controller. Call 'SplashScreen.show' for given view controller first."** which was caused by bad interaction of `expo-updates` and `expo-splash-screen` as mentioned [here](https://github.com/expo/expo/issues/7689#issuecomment-618133693).

I'd appreciate if you could update docs of installing `expo-updates` on bare workflow with `expo-splash-screen` while using `expo-modules`.

### Link to the related docs page

https://docs.expo.dev/bare/installing-updates/

**Comment by homosape** [comment]
Hi @amandeepmittal, could you please let me know if there is any update regarding this issue? 

I still receive error **No native splash screen registered for given view controller. Call 'SplashScreen.show** and I believe it is because Expo Splash Screen and Expo Updates don't work well together. 

I'm not sure how to fix this while using expo-modules environment

**Comment by amandeepmittal** [comment]
Hi @homosape, if you are using one of the latest Expo SDK in your bare workflow app, I'd suggest following the automatic installation steps for `expo-modules` instead of going through the route of using `react-native-unimodules` and setting up everything manually.

Link: https://docs.expo.dev/bare/installing-expo-modules/#automatic-installation

In case (for some reason the automatic installation doesn't work), it is still required on your end to follow manual steps, I'd suggest going through docs for that from here: https://docs.expo.dev/bare/installing-expo-modules/#manual-installation.

**Comment by homosape** [comment]
@amandeepmittal  when I initialized expo app it already had `expo-modules` this issue is more about implementing `expo-updates` and `expo-splash-screen` in bare workflow because I keep getting  No native splash screen registered for given view controller. Call 'SplashScreen.show' for given view controller first. but I am unable to implement [this line ](https://github.com/expo/expo/issues/7689) because it's using `react-native-unimodules`...

**Comment by amandeepmittal** [comment]
What steps are you following to initialize the Expo project? Can you please share the SDK version you are using on your project?

I do not see the steps you were referring to mentioned in the package's readme doc for the latest SDK. Have you checked the latest docs here: https://github.com/expo/expo/tree/main/packages/expo-updates#installation-in-bare-react-native-projects? They point to this doc for`expo-updates` in a bare React Native project: https://docs.expo.dev/bare/installing-updates/

**Comment by homosape** [comment]
I've initialised app with `npx create-expo-app`. I was a managed workflow SDK 45. Right now I am on SDK 47 and have ejected from managed workflow a while ago. I already had `expo-updates` and `expo-splash-screen` installed before I ejected.

I keep getting that splash screen warning message but only after I deliver OTA updates using `expo-updates`. I can include here more details if it can help. Just let me know what you might need.
